nrf_modem/doc/api.rst

```diff
@@ -262,3 +262,8 @@ OS specific definitions
 ***********************
 
 .. doxygengroup:: nrf_modem_os
+
+OS RPC transport definitions for nRF92
+**************************************
```

Comment:
```suggestion
OS RPC transport definitions for the nRF9230 SoC
************************************************
```
or may be just "OS RPC transport definitions"

Reply:
Thanks. I want to make it clear that this API is not related to nRF91xx. So I think having `nRF9230 SoC` in the title will lead to less confusion.